Documentation/contributor-guide/roadmap.md

```diff
@@ -17,16 +17,16 @@ For a full list of tasks in `v3.6.0`, please see [milestone etcd-v3.6](https://g
 
 | Title                                                                                                              | Priority | Status      | Note                                                                                                         |
```

Comment:
Note that we shouldn't use a markdown for project management. It's good to unify the names of priorities, however we don't want the roadmap to be overly specific

Reply:
My personal view is we should double down on the use of:
 - Github milestones for overall minor release tracking, and
 - Github projects for high level roadmap tracking.

However this roadmap markdown is ok for now, we just need to make sure it really does track only the genuinely major items for each release. Agree we should not add any more detail in the table otherwise it will be annoying to keep updating it.